src/models/domain/referenceEventData.ts

```diff
@@ -0,0 +1,5 @@
+export class ReferenceEventData {
```

Comment:
Per server PR comments, this should probably be moved and renamed as a API request model.

Reply:
Done, hopefully 🤞